README.md

```diff
@@ -258,18 +258,21 @@ If you have any questions about using mypy or types, please ask
 in the typing gitter instead: https://gitter.im/python/typing
 
 
-mypy_mypyc
-----------
+Compiled version of mypy
+------------------------
 
-We have built an experimental compiled version of mypy using the
-[mypyc compiler](https://github.com/mypyc/mypyc) for mypy-annotated
-Python code. It is approximately 4 times faster than interpreted mypy.
+We have built an compiled version of mypy using the [mypyc
+compiler](https://github.com/mypyc/mypyc) for mypy-annotated Python
+code. It is approximately 4 times faster than interpreted mypy and is
+available (and the default) for 64-bit Windows, OS X, and Linux.
```

Comment:
OS X is now officially called macOS, as far as I know

Reply:
ew